sdk/confidentialledger/azure-security-confidentialledger/README.md

```diff
@@ -150,6 +150,15 @@ This project has adopted the [Microsoft Open Source Code of Conduct][coc]. For m
 [jdk_link]: https://docs.microsoft.com/java/azure/jdk/?view=azure-java-stable
 [package]: https://mvnrepository.com/artifact/com.azure/azure-security-confidentialledger
 [samples_readme]: https://github.com/Azure/azure-sdk-for-java/tree/main/sdk/confidentialledger/azure-security-confidentialledger/src/samples/README.md
+[low_level_client]: https://github.com/Azure/azure-sdk-for-java/wiki/Low-Level-Client
```

Comment:
This page does not exist at present.

Reply:
The `Low-Level client` in track2 has been abandoned. so we think it is more appropriate to modify the description of `Low-Level client` in the document to `Http client`. Do you think this is okay?

Reply:
The description related to `HTTP client` has been removed.